core/src/main/resources/jenkins/model/Messages_fr.properties

```
@@ -21,28 +21,42 @@
 # THE SOFTWARE.
 
 Hudson.BadPortNumber=Numéro de port incorrect {0}
-Hudson.Computer.Caption=Maître
-Hudson.Computer.DisplayName=maître
+Hudson.Computer.Caption=Contrôleur
+Hudson.Computer.DisplayName=contrôleur
+Hudson.Computer.IncorrectNumberOfExecutors=Champ "nombre d''éxecuteurs" incorrect. Un nombre non négatif est attendu.
```

```suggestion
Hudson.Computer.IncorrectNumberOfExecutors=Champ "nombre d''éxecuteurs" incorrect. Un nombre positif est attendu.
```
Subjective, seems less complicated to me. (nit)

---

https://github.com/jenkinsci/jenkins/pull/8577/commits/9798d9a507b945d84a63aba26fa1ecb7d757d3ad